.github/workflows/update-parent-version.yaml

```diff
@@ -1,5 +1,5 @@
 # @author Ivan Senic
-name: Update parent version
+name: Update Parent Version
```

Comment:
Should name include "Main" somewhere, as it applies to main branch? (since this is shown in UI)

So like

    name: Update Parent Version (main)

Reply:
But this is jsonapi, there is only main :)

Reply:
Ah. Right, got confused. :)